toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -95,6 +95,19 @@ done
 [[ -z "${mode}" ]] && mode="build"
 [[ -z "${version}" ]] && version="2.0"
 
+# Set relevant folder definitions using Makefile that can be overriden by user
```

Comment:
This will work well if we are just pointing the tool at a non-standard path, but I'm curious if the user sets `SPECS_DIR=/path/to/my/specs` when calling the container tool if it will inherit here. When the recursive call to make happens I'm curious what environment it sees (since we were called from make, does it inherit it?)